src/routes/posts.js

```diff
@@ -18,15 +18,25 @@ const fetchPosts = async (req, res, next) => {
 
 		res.status(200).json({
 			status: 'success',
-			data: feed.map((article) => ({
-				id: article.id,
-				userId: article.userId,
-				title: article.title,
-				content: article.content,
-				image: article.image,
-				published: article.published,
-				createdOn: article.createdAt
-			}))
+			data: feed.map((post) => {
+				if(post.type === 'article')
+					return {
+						id: post.id,
```

Comment:
@beblicarl, I realize the fields don't seem to match what is in the document, intentional?

Reply:
> @beblicarl, I realize the fields don't seem to match what is in the document, intentional?

if it is the `userId` being used in place of the `authorId` then yes, i wanted to keep consistency with all the other endpoints

Reply:
@beblicarl  okay, the issue would have been an excellent place to voice the suggestion since we're in a team

and also I don't yet understand how it helps with consistency

Reply:
and its not just about the `userId`, there's others that don't match the doc

Reply:
> and its not just about the `userId`, there's others that don't match the doc

alright, i have made the change and pushed them to github